squidDrone/src/i2c/i2c_interface.hpp

```diff
@@ -0,0 +1,47 @@
+#ifndef SRC_I2C_INTERFACE_HPP_
+#define SRC_I2C_INTERFACE_HPP_
+
+#include "i2c_status.hpp"
+#include <stdint.h>
+#include <vector>
+#include <tuple>
+
+namespace i2c {
+
+  /**
+   * @brief Interface of I2C package
+   * 
+   */
+  class I2CInterface {
+    public:
+      virtual ~I2CInterface() = default;
+      explicit I2CInterface(void){};
+
+      /**
+       * @brief Reads data from I2C Bus
+       * 
+       * @param address The address of the I2C Bus participant
```

Comment:
what is the range of addresses you are testing against

Reply:
Same as above